modules/cluster-wide-proxy-updates.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-wide-proxy-updates_{context}"]
+= Updating your cluster-wide proxy
+
+After configuring your cluster-wide proxy, you can perform additional configurations on the proxy or add an Additional Trust Bundle.
```

Comment:
Sentence feels a little not right.. these steps are relevant if either:

- the cluster hasn't yet been configured with a proxy, but the cluster owner wants it to be
- the cluster has already been configured with a proxy, but now needs to be updated (for the reasons listed in the subsequent paragraph)